amazon-arm64.pkr.hcl

```diff
@@ -0,0 +1,205 @@
+variable "ami" {
+  type    = string
+  default = "ubuntu/images/hvm-ssd/ubuntu-focal-20.04-arm64-server-*"
+}
+
+variable "ami_name" {
+  type    = string
+  default = "supabase-postgres-arm-13.3.0.4"
+}
+
+variable "ami_regions" {
+  type    = list(string)
+  default = ["us-west-2"]
+}
+
+variable "ansible_arguments" {
+  type    = string
+  default = "--skip-tags,install-postgrest,--skip-tags,install-pgbouncer,--skip-tags,install-supabase-internal,ebssurrogate_mode='true'"
+}
+
+variable "aws_access_key" {
+  type    = string
+  default = ""
+}
+
+variable "aws_secret_key" {
+  type    = string
+  default = ""
+}
+
+variable "environment" {
+  type    = string
+  default = "prod"
+}
+
+variable "region" {
+  type    = string
+  default = "us-west-2"
+}
+
+variable "build-vol" {
+  type    = string
+  default = "xvdc"
+}
+
+# ccache docker image details
+variable "docker_user" {
+  type    = string
+  default = ""
+}
+
+variable "docker_passwd" {
+  type    = string
+  default = ""
+}
+
+variable "docker_image" {
+  type    = string
+  default = "laks/ccache-arm64"
```

Comment:
Sounds like this can be any docker image-can we just use an upstream alpine (or ubuntu) image?

Reply:
Yes, this can be any image.  one thing to note:  you need to `push`  for this image. Any ubuntu/alpine image from personal namespace should work. `(<dockerhub-username>/ubuntu)`